src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for Phone, Name, Email and Address class in the addressbook 
```

Comment:
It is best not to mention the classes using the interface. In future other classes might use it and your header comment needs to be updated every single time.